x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/home/hooks/use_available_packages.tsx

```diff
@@ -105,11 +105,23 @@ const packageListToIntegrationsList = (packages: PackageList): PackageList => {
         })
       : [];
 
-    const tiles = filterPolicyTemplatesTiles<PackageListItem>(
+    let tiles = filterPolicyTemplatesTiles<PackageListItem>(
       pkg.policy_templates_behavior,
       topPackage,
       integrationsPolicyTemplates
     );
+
+    // remove tiles that only have data streams of excluded types
+    tiles = tiles.filter((tile) => {
```

Comment:
This logic filters out those integrations that don't have any data streams, e.g. `aws.dynamodb` which only has metrics.

<img width="1181" height="962" alt="image" src="https://github.com/user-attachments/assets/d131dc7d-921e-4a98-8002-5e31ef7d130e" />